app/src/main/java/org/coepi/android/ui/alerts/AlertsFragment.kt

```diff
@@ -26,9 +26,18 @@ class AlertsFragment : Fragment() {
 
         toolbar.setNavigationOnClickListener { viewModel.onBack() }
 
-        val alertsAdapter = AlertsAdapter(onAlertClick = {
-            viewModel.onAlertClick(it)
-        }, alertsRepo = viewModel.alertsRepo)
+        val alertsAdapter = AlertsAdapter(
+            onAlertClick = {
+                viewModel.onAlertClick(it)
+            },
+            onAlertDismissed = {
+                viewModel.onAlertDismissed(it)
+            }
+        )
+
+        // clear system notifications if any are present
+        // TODO: will need to clear by unique notification ids
+        viewModel.onClearNotifications()
```

Comment:
This should be called something like `onUIReady()` (and put near to the end of `onCreateView` to be what the name says and potentially be usable by other logic). Calling it "onClearNotifications" is putting logic in the view.